src/components/training/LiveWorkoutMode.jsx

```
@@ -0,0 +1,1317 @@
+                >
+                  Remove Set
+                </Button>
+              </div>
+            )}
+
+            {/* Exercise Notes */}
+            <div className="mt-6">
+              <label className="block text-zinc-300 font-semibold mb-2">
+                Exercise Notes
+              </label>
+              <textarea
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)